x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -1062,3 +1069,14 @@ function getSearchSourceMigrations(
   }
   return filteredMigrations;
 }
+
+function removeIsSnoozedUntil(
+  doc: SavedObjectUnsanitizedDoc<RawRule>
+): SavedObjectUnsanitizedDoc<RawRule> {
+  return {
+    ...doc,
+    attributes: {
+      ...(omit(doc.attributes, ['isSnoozedUntil']) as RawRule),
```

Comment:
I believe the result of this fix is that a rule that was snoozed in 8.3 will no longer be snoozed in 8.4.1.

Given the urgency I'm 👍 on that, but we should probably make that clear in the 8.4.1 release notes.